templates/logstash.cnf.tpl

```diff
@@ -110,7 +110,7 @@ keyUsage = nonRepudiation, digitalSignature, keyEncipherment
 subjectKeyIdentifier=hash
 authorityKeyIdentifier=keyid:always,issuer
 basicConstraints = critical,CA:true
-subjectAltName= IP: "{{ elk_ip }}"
+subjectAltName= IP: "{{ elk_url }}"
```

Comment:
add default value to 
`defaults/main.yml`
ex. `127.0.0.1`

Reply:
elk_url it's not local IP, like 127.0.0.1. it's our external IP address. Anyway add our external IP to defaults?

Reply:
But we can't write external IP. It's only for role.